Source/IGListAdapter.m

```diff
@@ -1015,6 +1015,25 @@ - (void)scrollToSectionController:(IGListSectionController<IGListSectionType> *)
     [self.collectionView scrollToItemAtIndexPath:indexPath atScrollPosition:scrollPosition animated:animated];
 }
 
+- (void)invalidateLayoutForSectionController:(IGListSectionController<IGListSectionType> *)sectionController {
+    IGAssertMainThread();
+    IGParameterAssert(sectionController != nil);
+    
+    NSInteger itemCount = [sectionController numberOfItems];
```

Comment:
Don't mind adding this, but for the sake of learning -- Is putting const here the same as using 'let' in Swift? "A non mutable version"